Externals.cfg

```diff
@@ -2,8 +2,7 @@
 protocol = git
 repo_url = https://github.com/ufs-community/regional_workflow
 # Specify either a branch name or a hash but not both.
-#branch = develop
-hash = e5d5c38
+hash = f5f8158
 local_path = regional_workflow
```

Comment:
@mark-a-potts, can you push a quick commit to add the commented branch here?  We use that for all external repos in case users want to reference a branch instead.  Thanks.

Reply:
Done.

Reply:
Sorry for the confusion.  The branch line would still be commented, but we'd point to a hash of the regional_workflow repo.  The latest is [a3e0066](https://github.com/ufs-community/regional_workflow/commit/a3e006636005cbad7974065d3692b92fd722c2fe), so it would be:

# branch = develop
hash = a3e0066